doc/source/guide/data_types/maps.rst

```diff
@@ -137,28 +151,330 @@ more complicated figure.
+property. In the following example we change the title of the plot by changing the
+`~sunpy.map.GenericMap.plot_settings` property.
+
+.. plot::
+
+    import sunpy.map
+    import sunpy.data.sample
+    import matplotlib.pyplot as plt
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    smap.plot_settings['title'] = "My Second SunPy Plot"
+    smap.plot_settings['cmap'] = plt.cm.Blues_r
+    fig = plt.figure()
+    smap.plot()
+    plt.colorbar()
+    plt.show()
+
+
+8. Colormaps and Normalization
+******************************
+
+Image data is generally shown in false color in order to better identify it or
+to better visualize structures in the image. Matplotlib handles this colormapping
+process through the `~matplotlib.colors` module. This process involves two steps:
+the data array is first mapped onto the range 0-1 using an instance of
+`~matplotlib.colors.Normalize` or a subclass; then this number is mapped to a
+color using an instance of a subclass of a `~matplotlib.colors.colormap`.
+
+SunPy provides the colormaps for each mission as defined by the mission teams.
+The Map object chooses the appropriate colormap for you when it is created as
+long as it recognizes the instrument. To see what colormaps are available::
+
+    import sunpy.cm
+    sunpy.cm.cmlist.keys()
+
+The SunPy colormaps are registered with matplotlib so you can grab them like
+you would any other colormap::
+
+    import matplotlib.pyplot as plt
+    import sunpy.cm
+    # You need to import sunpy.cm or sunpy.map for this to work.
+    cmap = plt.get_cmap('sdoaia171')
+
+
+The following plot shows off all of the colormaps.
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import sunpy.cm
+    sunpy.cm.show_colormaps()
+
+These can be used with the standard commands to change the colormap. So for
+example if you wanted to plot an AIA image but use an EIT colormap, you would
+do so as follows.
+
+.. plot::
+
+    import sunpy.map
+    import sunpy.data.sample
+    import matplotlib.pyplot as plt
+
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    cmap = plt.get_cmap('sohoeit171')
+
+    fig = plt.figure()
+    ax = plt.subplot(1,1,1)
+    smap.plot(cmap=cmap)
+    plt.colorbar()
+    plt.show()
+
+or you can just change the colormap for the map itself as follows::
+
+    cmap.plot_settings['cmap'] = plt.get_cmap('sohoeit171')
+
+The normalization is also set automatically and is chosen so that all the
+data from minimum to maximum is displayed as best as possible for most cases.
+This means that it is never necessary to touch the data such as applying a function
+such sqrt or log to the data to make your plot look good.
+There are many normalizations available from matplotlib such as `~matplotlib.colors.Lognorm`, or
+`~matplotlib.colors.PowerNorm`. Other
+`more exotic normalizations <http://physics.mnstate.edu/craig/apy10/visualization/index.html>`_ are also
```

Comment:
this link should point to the main astropy docs: http://docs.astropy.org/en/stable/visualization/index.html